ad9361/sw/ad9361_api.c

```diff
@@ -1424,7 +1424,7 @@ int32_t ad9361_get_tx_lo_freq (struct ad9361_rf_phy *phy,
 	*lo_freq_hz = ad9361_from_clk(clk_get_rate(phy,
 										phy->ref_clk_scale[TX_RFPLL]));
 
-	return 0;
+	return 0ad9361_set_tx_lo_int_ext;
 }
```

Comment:
Thanks for fixing the issues.

Can you resend with this line fixed? Thanks.

Reply:
I resubmitted after removing the accidentally pasted "0ad9361_set_tx_lo_int_ext" on line 1427.